assets/prototype/domain/eventEditor/tickets/ticketPriceCalculator/hooks/calculateTicketTotal.ts

```diff
@@ -0,0 +1,36 @@
+import { filter, pathOr, propSatisfies, reduce } from 'ramda';
+
+import ticketTotalCalculator from './ticketTotalCalculator';
+import { TpcFormData } from '../types';
+import { Price, Ticket } from '../../../data/types';
+import { sortByPriceOrderIdAsc } from '../../../../shared/predicates/prices/sortingPredicates';
+import { updateTicketPriceForTicket } from '../../../../shared/predicates/tickets/selectionPredicates';
+
+const notNewPrice = propSatisfies((id) => id !== 'NEW_PRICE', 'id');
+const NOT_A_TICKET = { id: '' };
+const EMPTY_ARRAY = [];
+
+const calculateTicketTotal = (state: TpcFormData): TpcFormData => {
+	const ticket = pathOr<Ticket | { id: '' }>(NOT_A_TICKET, ['ticket'], state);
```

Comment:
I am not happy with `| { id: '' }` 😄

Reply:
what options do I have as an alternative?

Reply:
`const ticket = pathOr<Ticket>(null, ['ticket'], state);`